code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -248,6 +259,8 @@
 	body_parts_covered = 0
 	var/mob/observer/eye/aiEye/eye
 	spawn_blacklisted = TRUE
+	style_coverage = COVERS_WHOLE_FACE
+	style = STYLE_NEG_HIGH
```

Comment:
Mask actually looks nice, but it's broken and does not spawn anymore so change is pointless.